apps/wallet/src/background/accounts/zk/utils.ts

```diff
@@ -130,8 +130,7 @@ type WalletInputs = {
 
 export type PartialZkSignature = Omit<ZkSignatureInputs, 'addressSeed'>;
 
-// TODO: update when we have the final production url (and a https one)
-const zkProofsServerUrl = 'http://prover-devnet.mystenlabs.com:8080';
+const zkProofsServerUrl = 'https://prover.mystenlabs.com/v1';
```

Comment:
this is now the final prod url see https://mysten-labs.slack.com/archives/C05Q4FNS31A/p1695137156745129